resources/js/components/shared/ModalSaveVersion.vue

```diff
@@ -306,10 +307,60 @@ export default {
     });
     this.retrieveSavedSearchCharts();
     this.getProcessDescription();
+
     //Receives selected Option from launchpad Icons multiselect
     this.$root.$on("launchpadIcon", this.launchpadIconSelected);
   },
   methods: {
+    /**
+     * Get all information related to Launchpad Settings Modal
+     */
+    getLaunchpadSettings() {
+      this.images = [];
+      ProcessMaker.apiClient
+        .get("processes/" + this.processId + "/media")
+        .then((response) => {
+          const launchpadProperties = JSON.parse(
+            response.data.data[0]?.launchpad_properties
+          );
+
+          if (launchpadProperties && Object.keys(launchpadProperties).length > 0) {
+            this.selectedSavedChart = launchpadProperties.saved_chart_title
+              ? launchpadProperties.saved_chart_title
+              : "";
+            this.selectedSavedChartId = launchpadProperties.saved_chart_id;
+            this.selectedLaunchpadIcon = launchpadProperties.icon;
+            this.selectedLaunchpadIconLabel = launchpadProperties.icon_label;
+          } else {
+            this.selectedSavedChart = "";
```

Comment:
this is a string or object,
in the data section is defined as Object

Reply:
Solved. It was changed to String in data section